File: Aug16/Troy/Q5.java
class Solution {
    public void moveZeroes(int[] nums) {
        int len = nums.length;
        int numOfZeros = 0;
        for (int i = 0; i < len; i++) {
            if (nums[i] == 0) {
                numOfZeros++;
            } else {
                nums[i - numOfZeros] = nums[i];
            }

        }

        for (int j = len - 1; j >= len - numOfZeros; j--) {
            nums[j] = 0;
        }
    }
}

File: Aug14/Carl/Q5.py
class Solution:
    def arrayPairSum(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        nums = sorted(nums)
        totalSum = 0
        for i in range(0, len(nums), 2):
            totalSum += nums[i]
        return totalSum


S = Solution()
print(S.arrayPairSum([1, 5, 3]))
